instaclustr/structs.go

```diff
@@ -112,28 +113,29 @@ type DataCentreCreateRequest struct {
 }
 
 type AddonBundles struct {
-	Bundle  string `json:"bundle"`
-	Version string `json:"version"`
+	Bundle  string        `json:"bundle" mapstructure:"bundle"`
+	Version string        `json:"version" mapstructure:"version"`
+	Options BundleOptions `json:"options,omitempty" mapstructure:"options"`
```

Comment:
Would be better to have an AddonBundleOptions struct, unless there's a reason we can't easily switch to that?

Reply:
Nah, can easily switch to that.

Reply:
Actually can't easily switch to that. When read, we have a `addOnBundles` but when creating they are all `bundle` and share the same type. So separating them will be a bit clunky since there are 2 options struct to maintain. In this case, everything in AddonBundleOptions will need to be included in BundleOptions as well.